webapp/packages/plugin-user-profile/src/UserProfileService.ts

```diff
@@ -47,15 +47,15 @@ export class UserProfileService {
     }
   }
 
-  async open(): Promise<boolean> {
+  open = async (): Promise<boolean> => {
     const state = await this.optionsPanelService.open(formGetter);
 
     if (state) {
       this.change();
     }
 
     return state;
-  }
+  };
```

Comment:
Please revert. Arrow functions are hard to debug, and please avoid arrow function usage in services